pyriemann/utils/covariance.py

```diff
@@ -117,27 +159,30 @@ def _check_est(est):
     return est
 
 
-def covariances(X, estimator='cov'):
+def covariances(X, estimator='cov', **kwds):
     """Estimation of covariance matrix.
 
     Parameters
     ----------
     X : ndarray, shape (n_matrices, n_channels, n_times)
         Multi-channel time-series.
-    estimator : {'corr', 'cov', 'lwf', 'mcd', 'oas', 'sch', 'scm'}, \
+    estimator : {'corr', 'cov', 'fpcm', 'lwf', 'mcd', 'oas', 'sch', 'scm'}, \
             default='scm'
-        Covariance matrix estimator [1]_:
-
-        * 'corr' for correlation coefficient matrix [2]_,
-        * 'cov' for numpy based covariance matrix [3]_,
-        * 'lwf' for shrunk Ledoit-Wolf covariance matrix [4]_,
-        * 'mcd' for minimum covariance determinant matrix [5]_,
-        * 'oas' for oracle approximating shrunk covariance matrix [6]_,
-        * 'sch' for Schaefer-Strimmer covariance matrix [7]_,
-        * 'scm' for sample covariance matrix [8]_,
+        Covariance matrix estimator [est]_:
+
+        * 'corr' for correlation coefficient matrix [corr]_,
+        * 'cov' for numpy based covariance matrix [cov]_,
+        * 'fpcm' for fixed point covariance matrix [fpcm]_,
```

Comment:
I find the name not super informative and reading the PR it's not super clear what it is useful for and why I should use it? Can you add a blob of text somewhere to explain this?

🙏